Makefile

```diff
@@ -162,7 +168,16 @@ help:
 	@echo "         make TARGET WORKDIR=/tmp "
 	@echo ""
 	@echo "- REPO_URL"
-	@echo "use it to change the where rootfs base packages will be downloaded"
+	@echo "Use it to change the where rootfs base packages will be downloaded."
+	@echo ""
+	@echo "- IMG_SIZE"
+	@echo "Change the image size of the image to generate."
```

Comment:
It would be useful to know what units this variable is expecting so I'd add something like, " (accepts any value recognised by qemu-img(1))."

Reply:
done.